### PR TITLE
Add WCP_Namespaced_Class_And_Windows_Support FSS into VMOP

### PR DESCRIPTION
**Pull request by zyiyi11**
This MR adds WCP_Namespaced_Class_And_Windows_Support FSS into VMOP. It will guard both VM Serive migration of VM Class to namespace scope work and Windows support work.

Testing:
- Vmop: https://buildweb.eng.vmware.com/sb/63377089/
- Csp: https://buildweb.eng.vmware.com/sb/63377286/
- dev-integ-vds: https://wcp.svc.eng.vmware.com/job/dev-integ-vds/3070/
```
root@42037808056efc7c1853467a8ae6218b [ ~ ]#  k describe deployment vmware-system-vmop-controller-manager -n vmware-system-vmop | grep "FSS_WCP_NAMESPACED_CLASS_AND_WINDOWS_SUPPORT"
      FSS_WCP_NAMESPACED_CLASS_AND_WINDOWS_SUPPORT:  false
```

### CODE REVIEW

File: pkg/lib/env.go [hunk 1]
```diff
@@ -97,6 +98,10 @@ var IsWCPVMImageRegistryEnabled = func() bool {
 	return os.Getenv(VMImageRegistryFSS) == trueString
 }
 
+var IsVMClassMigrateNamespaceFSSEnabled = func() bool {
```
Comment (cyclexuxu) [inline pkg/lib/env.go:101]: Can we change the name to be consistent with the FSS name?

Comment (zyiyi11) [reply pkg/lib/env.go:101]: Right. Thx for catching that.

File: hack/deploy-wcp.sh [hunk 2]
```diff
@@ -35,6 +35,9 @@ FSS_WCP_FAULTDOMAINS_VALUE=${FSS_WCP_FAULTDOMAINS_VALUE:-false}
 # Image Registry: we use this FSS for VM publish.
 FSS_WCP_VM_IMAGE_REGISTRY_VALUE=${FSS_WCP_VM_IMAGE_REGISTRY_VALUE:-false}
 
+# VM Class migration to namespace FSS
```
Comment (cyclexuxu) [inline hack/deploy-wcp.sh:38]: nit: maybe also add `and windows support`?

**Review by cyclexuxu — APPROVED** [review]
LGTM.

**Pull request merged**